hw/opentitan/ot_keymgr.c

```diff
@@ -83,6 +83,8 @@ static_assert(KEYMGR_GEN_DATA_BYTES <= KEYMGR_KDF_BUFFER_BYTES,
 #define KEYMGR_ENTROPY_WIDTH  (KEYMGR_LFSR_WIDTH / 2u)
 #define KEYMGR_ENTROPY_ROUNDS (KEYMGR_KEY_WIDTH / KEYMGR_ENTROPY_WIDTH)
 
+#define KEYMGR_LFSR_SEED_BYTES ((KEYMGR_LFSR_WIDTH) / 8u)
```

Comment:
maybe add a static assert to make sure that `KEYMGR_LFSR_SEED_BYTES <= KEYMGR_SEED_BYTES`? (because `seeds` array is allocated with string of length `KEYMGR_SEED_BYTES`)